edgelet/build/linux/package.sh

```diff
@@ -0,0 +1,499 @@
+        else
+            RPM_RELEASE="$REVISION"
+        fi
+
+        CMAKE_ARGS="$CMAKE_ARGS -DCPACK_GENERATOR=RPM"
+        CMAKE_ARGS="$CMAKE_ARGS '-DCPACK_PACKAGE_VERSION=$RPM_VERSION'"
+        CMAKE_ARGS="$CMAKE_ARGS '-DCPACK_RPM_PACKAGE_RELEASE=$RPM_RELEASE'"
+        ;;
+
+    *)
+        CMAKE_ARGS="$CMAKE_ARGS -DCPACK_GENERATOR=DEB"
+        CMAKE_ARGS="$CMAKE_ARGS '-DCPACK_PACKAGE_VERSION=$VERSION'"
+        CMAKE_ARGS="$CMAKE_ARGS '-DCPACK_DEBIAN_PACKAGE_RELEASE=$REVISION'"
+        ;;
+esac
+
+
+case "$PACKAGE_OS" in
+    'centos7')
+        case "$PACKAGE_ARCH" in
+            'amd64')
+                DOCKER_IMAGE='centos:7.5.1804'
+                ;;
+
+            # CentOS 7's base repo does not have cross-compiler packages.
+            #
+            # EPEL does have cross-compiler gcc and g++ packages, but not libc.
+            # The general intent of upstream is that the cross-compilers are only used for kernel development.
+            #
+            # Fedora's repo does have cross-compiler libc as well, but it may not be usable
+            # because of https://bugzilla.redhat.com/show_bug.cgi?id=1456209
+            #
+            # So stick with the linaro compiler for now.
+
+            'arm32v7')
+                DOCKER_IMAGE='edgebuilds.azurecr.io/gcc-linaro-7.3.1-2018.05-x86_64_arm-linux-gnueabihf:centos_7.5-1'
+                ;;
+
+            'aarch64')
+                DOCKER_IMAGE='edgebuilds.azurecr.io/gcc-linaro-7.3.1-2018.05-x86_64_aarch64-linux-gnu:centos_7.5-1'
+                ;;
+        esac
+
+        CMAKE_ARGS="$CMAKE_ARGS '-DOPENSSL_DEPENDS_SPEC=openssl-libs'"
+        ;;
+
+    'debian8')
+        case "$PACKAGE_ARCH" in
+            'amd64')
+                DOCKER_IMAGE='debian:8-slim'
+                ;;
+
+            # Debian 8 doesn't have cross-compiler packages in its main repo
+            #
+            # The emdebian repos have cross-compiler packages but they cannot be installed because of broken dependencies.
+            # >gcc-4.9-arm-linux-gnueabihf : Depends: libgcc-4.9-dev:armhf (= 4.9.2-10) but 4.9.2-10+deb8u2 is to be installed
+            #
+            # emdebian is also not maintained any more, not in the least because Debian 9+ have cross-compiler packages in the main repo.
+            #
+            # So stick with the linaro compiler for now.
+            'arm32v7')
+                DOCKER_IMAGE='edgebuilds.azurecr.io/gcc-linaro-7.3.1-2018.05-x86_64_arm-linux-gnueabihf:debian_8.11-1'
+                ;;
+
+            'aarch64')
+                # Like the comment in packages.yaml says, Debian 8 aarch64 is not LTS, and doesn't have any official or ports repos.
+                #
+                # So don't build it at all.
+                ;;
+        esac
+
+        CMAKE_ARGS="$CMAKE_ARGS '-DOPENSSL_DEPENDS_SPEC=libssl1.0.0'"
+        ;;
+
+    'debian9')
+        DOCKER_IMAGE='debian:9-slim'
+        CMAKE_ARGS="$CMAKE_ARGS '-DOPENSSL_DEPENDS_SPEC=libssl1.1'"
```

Comment:
>1.1 [](start = 62, length = 3)

This is a change from the previous build.  Was this on purpose?

Reply:
Yeah. I didn't think there was a reason to keep it on 1.0